### PR TITLE
SAM-2687 Adding elFinder support to samigo

### PR DESCRIPTION
**Pull request by ddelblanco**
I feel there are some things we can improve in the future here (as show or not this option based in some permissions and add the i18n. But this is the basic functionality working and we can improve that in next jiras.

### CODE REVIEW

File: textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/samigo/SamFsItem.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.sakaiproject.elfinder.sakai.samigo;
```
Comment (master-bob) [inline textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/samigo/SamFsItem.java]: No Comment header?

Comment (ddelblanco) [inline textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/samigo/SamFsItem.java]: Oh, that's true. I copied this from the other elfinder configurations. I can add that and update the PR. Thanks!

Comment (ddelblanco) [inline textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/samigo/SamFsItem.java]: I've updated the 2 files I've created with the license.

**Comment by ddelblanco** [comment]
@buckett Do you think we should merge this?

**Comment by juanjmerono** [comment]
Can you amend to add SAM-2687 at the beginning of your commit message?

**Comment by ottenhoff** [comment]
You can amend the commit message via Merge PR -> Confirm squash and merge

**Comment by ddelblanco** [comment]
Done (Hope not had broken anything in the road!)

**Pull request merged**